Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -131,57 +126,31 @@ The Leap Motion Hands will show up in the editor, without a VR headset plugged i
 
 ## Removing Leap Motion from the Project
 
+1. Navigate to the **Mixed Reality Toolkit** > **Leap Motion** > **Disconnect Leap Motion Unity Modules**
+    - Let Unity refresh as references in the **Microsoft.MixedReality.Toolkit.Providers.LeapMotion.asmdef** file are modified in this step
 1. Close Unity
 1. Close Visual Studio, if it's open
 1. Open File Explorer and navigate to the root of the MRTK Unity project
     - Delete the **UnityProjectName/Library** directory
-    - Delete the **UnityProjectName/Assets/LeapMotion** directory
-    - Delete the **UnityProjectName/Assets/LeapMotion.meta** file
+    - Delete the **UnityProjectName/Assets/Plugins/LeapMotion** directory
```

Comment:
Is this a path change for 4.5.1? Or is it also the path for 4.5.0?

Reply:
The path for the 4.4.0 assets was the previous path and then for the 4.5.0 and 4.5.1, the path was changed to include Plugins